scripts/health/dynamic_text.py

```diff
@@ -0,0 +1,75 @@
+""" """
+import pandas as pd
+import json
+from scripts.config import PATHS
+from scripts.owid_covid import tools as owid_tools
+from bblocks.import_tools.world_bank import WorldBankData
+import country_converter as coco
+from scripts.health.common import get_malaria_data
+
+
+def spending_dynamic():
+    """Create dynamic text for health spending"""
+    wb = WorldBankData()
+    wb.load_indicator('SH.XPD.CHEX.PC.CD')
+    wb.load_indicator('SH.XPD.CHEX.GD.ZS')
+
+    pc = (wb.get_data('SH.XPD.CHEX.PC.CD')
+          .dropna(subset='value')
+          .sort_values('date')
+          .groupby('iso_code', as_index=False)
+          .last()
+          .loc[:, ['iso_code', 'value']]
+          .assign(continent=lambda d: coco.convert(d.iso_code, to='continent', not_found=None))
+          .loc[lambda d: (d.continent == 'Africa') & (d.value >= 86), :]
+          .assign(indicator = 'pc')
+          )
+    gdp = (wb.get_data('SH.XPD.CHEX.GD.ZS')
+           .dropna(subset='value')
+           .sort_values('date')
+           .groupby('iso_code', as_index=False)
+           .last()
+           .loc[:, ['iso_code', 'value']]
+           .assign(continent=lambda d: coco.convert(d.iso_code, to='continent', not_found=None))
+           .loc[lambda d: (d.continent == 'Africa') & (d.value >= 5), :]
+           .assign(indicator = 'gdp')
+           )
+
+    dff = pd.concat([pc, gdp])
+    dff = dff.pivot(index='iso_code', columns = 'indicator', values='value')
+    dff = dff.dropna()
+
```

Comment:
you should deal with getting/cleaning these indicators in separate function, and then have a function that brings them both together into the single df (re: single responsibility principle)